### PR TITLE
Added Python resources

### PR DESCRIPTION
**Pull request by jackmcbride**

### CODE REVIEW

File: python/resources.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Python Development Resources
             + [Codecademy](http://www.codecademy.com/tracks/python)
             + [Coursera](https://www.coursera.org/courses?search=python)
             + [Edx](https://www.edx.org/course/mitx/mitx-6-00-1x-introduction-computer-1841#.U5in4IaP8xA)
+            + [Udemy](https://www.udemy.com/the-python-mega-course/learn/v4/overview)
```
Comment (vicky002) [inline python/resources.md]: @jackmcbride This seems like a paid course. Now we are not adding paid course link, There are few in the repository and I'll remove them as well.

File: python/resources.md [hunk 2]
```diff
@@ -167,4 +169,7 @@ to manipulate ISZ files (.isz), including .isz to .iso conversion
     + [Webfaction](https://www.webfaction.com/) - Shared hosting
 + Books
     + [Python Algorithms: Mastering Basic Algorithms](http://www.amazon.in/Python-Algorithms-Mastering-Basic-Language/dp/148420056X/ref=dp_ob_title_bk) - Great Book for learning Basic Data Structures and Algorithms in Python (3.9 of 5 by GoodReads)
```
Comment (vicky002) [inline python/resources.md]: I'll Remove this.

File: python/resources.md [hunk 3]
```diff
@@ -167,4 +169,7 @@ to manipulate ISZ files (.isz), including .isz to .iso conversion
     + [Webfaction](https://www.webfaction.com/) - Shared hosting
 + Books
     + [Python Algorithms: Mastering Basic Algorithms](http://www.amazon.in/Python-Algorithms-Mastering-Basic-Language/dp/148420056X/ref=dp_ob_title_bk) - Great Book for learning Basic Data Structures and Algorithms in Python (3.9 of 5 by GoodReads)    
-    + [Distributed Computing with Python](https://www.packtpub.com/big-data-and-business-intelligence/distributed-computing-python) - A good beginner book for learning about Distrubted Computer Programming in Python
+    + [Distributed Computing with Python](https://www.packtpub.com/big-data-and-business-intelligence/distributed-computing-python) - A good beginner book for learning about Distributed Computer Programming in Python
```
Comment (vicky002) [inline python/resources.md]: This also, needs to be removed. Amazon Paid Book lInk. 

Many people add affiliate links from Amazon so now we have stopped adding paid books links as well.

**Comment by vicky002** [comment]
I will have to looking into this closely.  Seems like you were behind few commits and merged the whole branch into yours. 

This shouldn't really affect anything but I have to see all the changes once.

Give me some time please @jackmcbride 🙂

**Comment by vicky002** [comment]
@codingonHP can you please start looking into al the old PRs and merge them? 🙏 

If there are any conflicts, please try to fix or assign it to the user who has worked on this change.

**Comment by codingonHP** [comment]
@vicky002 Sure, let me see what I can do.

**Comment by codingonHP** [comment]
Hey @jackmcbride if you get time, please have a look at this :)

**Comment by codingonHP** [comment]
closing due to inactivity.

**Pull request closed without merging**